app/filters.js

```diff
@@ -9,58 +8,6 @@ module.exports = function (env) {
    */
   let filters = {}
 
-  /* ------------------------------------------------------------------
-    date filter for use in Nunjucks
-    example: {{ params.date | date("DD/MM/YYYY") }}
-    outputs: 01/01/1970
-  ------------------------------------------------------------------ */
-  filters.date = function(timestamp, format) {
```

Comment:
We shouldn't expose this. But I think you said you wanted to raise an issue and deal with this outside of this ticket?

Reply:
Yes, a logging an issue is preferable. We need to consider what formats are used, for example "DD/MM/YYYY" is not encapsulated, yet used.